generator/cmd/protoc-gen-gclient/main_test.go

```diff
@@ -26,7 +26,7 @@ func TestMain(m *testing.M) {
 }
 
 func TestRun_Rust(t *testing.T) {
-	if err := run("testdata/rust/rust.bin", "testdata/rust/golden", "../../templates"); err != nil {
+	if err := run(os.Stdin, os.Stdout, "testdata/rust/rust.bin", "testdata/rust/golden", "../../templates"); err != nil {
```

Comment:
nit: do we suspect to every use values different from stdin/stdout? I am curious why we should make them configurable?

Reply:
I found it surprising that `run` was reading/writing to a location that I didn't tell it to, so the reason was to make it explicit. 

I also imagine that at some point we will want to write [`outBytes`](https://github.com/googleapis/google-cloud-rust/pull/70/files#diff-847456702a1cf33aeec38dbdda1288d610dc4e8f6c926eb67dcc6a74124d22caR97) to somewhere other than stdout?

Reply:
I think that is just the contract for protoc. But if you think it adds value I am fine with it. If we ever add a test to validate output you are right it would be useful to capture.